client/blocks/subscription-length-picker/index.jsx

```diff
@@ -106,20 +138,26 @@ export class SubscriptionLengthPicker extends React.Component {
 	};
 }
 
-export function myFormatCurrency( price, code ) {
+export function myFormatCurrency( price, code, options = {} ) {
```

Comment:
Totally soft opinion:

Would there be a better name for this function? I know it's been here for quite a while, but it would be great if it can have a more catchy name fitting its semantics, or even integrate with the underlying `formatCurrency` as a configurable behavior.

Reply:
Yeah... It's dropping the cents if they're empty. `formatCurrency` uses the more general term 'fraction', so we could add an option named something like 'fractionOptional' or 'hideEmptyFraction'? `Intl.numberFormat` uses `minimumFractionDigits`, which isn't quite what we want (because we don't want to write `$12.3`, do we?).

How about this deal?: If you choose a good name for it, I'll move the functionality into a `formatCurrency()` option in a separate PR

Reply:
Sounds good :)